src/main/java/com/conveyal/taui/models/AddTripPattern.java

```diff
@@ -15,6 +15,9 @@ public String getType() {
         return type;
     }
 
+    // Using Integer because transitMode can be null
```

Comment:
Maybe expand comment to say why it can be null (backward compatibility, field didn't exist in the past)